guides/representative-selection.md

```diff
@@ -0,0 +1,60 @@
+# Representative selection process
+
+This document is a guide for the selection of Representatives for the Leadership Council. This gives recommendations for how a top-level team should select their Representative. It is not a requirement that teams follow this guide, and may choose their own process.
+
+This process is inspired by the [Sociacracy selection process](https://www.sociocracyforall.org/selection-process/).
+
+## Recommendations on candidate choice
+
+See [candidate criteria] for guidance for choosing a candidate. See [representative role] for a description of what a Representative is expected to do.
+
+Many of the same qualities that make a good Council representative also make a good team lead. However, we strongly recommend against individuals taking on both the role of team lead and Council representative simultaneously. In the past, our experience has shown that it is extremely difficult to balance the demands of both roles effectively. For teams with shared leadership responsibilities (e.g. co-leads), assigning one co-lead as the representative may be fine if the other co-lead(s) are prepared to take on more of the team leadership responsibilities. However, for teams with a single lead, we recommend either establishing multiple clearly delineated team leadership roles or selecting a different Council representative.
+
+Remember that Council representatives can be subteam members, if a subteam member has the requisite scope and skillset.
```

Comment:
```suggestion
Remember that Council representatives can be subteam members, if a subteam member has the requisite scope and skillset. For the avoidance of doubt, working groups count as subteams for this purpose.
```

Reply:
I added a complete definition of what is meant here.